com.playonlinux.PlayOnLinux4.yaml

```diff
@@ -0,0 +1,148 @@
+id: com.playonlinux.PlayOnLinux4
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.winehq.Wine
+base-version: stable-22.08
+command: playonlinux
+finish-args:
+  - --socket=x11
+  - --device=all
+  - --socket=pulseaudio
+  - --share=ipc
+  - --share=network
+  - --allow=multiarch
+  - --filesystem=host:rw
+  - --env=DEBIAN_PACKAGE=TRUE
```

Comment:
why do you need this. It feels misleading.

Reply:
See https://github.com/PlayOnLinux/POL-POM-4/blob/8dddc981035d705cd489272d2cc3d5c9d9f9427c/python/lib/Variables.py#L26-L29. I had this now moved to a Patch.